src/main/java/frc/robot/subsystems/MoveableSubsystem.java

```diff
@@ -4,16 +4,14 @@
 
 /**
  * This is interface for all of our moving subsystems with motors.
- * For example every moving SS will have to move, stop etc.
+ * For example every moving SS will have to move and stop.
```

Comment:
Good job removing unnecessary methods. I wanted to open an issue about this and then I found that this pull request fixes this